packages/react-scripts/scripts/build-module.js

```diff
@@ -1,58 +1,82 @@
 require('../utils/loadEnv');
 
-var paths = require('../config/paths');
+var path = require('path');
+var fs = require('fs-extra');
+var chokidar = require('chokidar');
+var transformFileSync = require('babel-core').transformFileSync;
 var spawn = require('cross-spawn');
-var result;
+var paths = require('../config/paths');
 
-var defaults = [
-  '--presets=trunkclub',
-  '--source-maps'
-];
+function lint () {
+  return spawn.sync('node', [require.resolve('./lint')], {stdio: 'inherit'});
+}
 
-var directoryArgs = [
-  paths.appSrc,
-  '--out-dir=' + paths.appBuild
-];
+function transformWithBabel (filePath) {
+  const { code } = transformFileSync(path.join(paths.appSrc, filePath), {
+    ast: false,
+    sourceMaps: 'inline',
```

Comment:
I guess I'm just thinking that inline source maps are not ok for node modules that are built, when if we're using the babel cli it will just output them separately for you instead of inline.